### PR TITLE
Remove selection rectangle, Closes #30

### LINKED ISSUE #30: Disable the rectangle select from appearing
Unit selection is still visible/occurs and is just confusing

### PR DESCRIPTION
**Pull request by TurBoss**
this also prompts to console a message that cmdcolors has been changed

### CODE REVIEW

**Comment by TurBoss** [comment]
;( why doesn work now???

File: LuaUI/widgets/gui_standalone_lobby.lua [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+		desc      = "Standalone lobby tools",
+		author    = "gajop",
+		date      = "in the future",
+		license   = "GPL-v2",
+		layer     = 1001,
+		enabled   = true,
+	}
 end
 
+spSendCommands = Spring.SendCommands
```
Comment (gajop) [inline LuaUI/widgets/gui_standalone_lobby.lua]: I deeply dislike localizing (Spring) function names like this. Write the whole thing.

Comment (TurBoss) [inline LuaUI/widgets/gui_standalone_lobby.lua]: does not localization speed up calls?

Comment (gajop) [inline LuaUI/widgets/gui_standalone_lobby.lua]: You should read about premature optimization, this is a prime example. The impact is minimal. 

I doubt it's actually even an improvement as you're allocating a new variable just for two invocations, which has to be deallocated later. Not to mention you're not making the variable local and thus polluting global scope.

The point is, these miniature optimizations are irrelevant and clear code should always be chosen instead.

**Comment by TurBoss** [comment]
can't figure out if i'm wrong

https://github.com/spring/spring/commit/3b9c03af3c58e3dbc6758b76afc71fba4d616971

**Comment by gajop** [comment]
You'll have to figure it out yourself.

**Comment by TurBoss** [comment]
i'm on it :D

**Comment by gajop** [comment]
Does this work right now?

**Comment by TurBoss** [comment]
yes is working

**Comment by gajop** [comment]
great, thanks

**Pull request merged**